toolchain/mfc/gen_physics_docs.py

```
@@ -0,0 +1,214 @@
+from pathlib import Path
+from typing import Dict, List, Set
+
+HERE = Path(__file__).resolve().parent
+CASE_VALIDATOR_PATH = HERE / "case_validator.py"
+
+# Make the toolchain package importable
+_toolchain_dir = str(HERE.parent)
+if _toolchain_dir not in sys.path:
+    sys.path.insert(0, _toolchain_dir)
+
+from mfc.case_validator import PHYSICS_DOCS  # noqa: E402  pylint: disable=wrong-import-position
```

Ruff (0.15.0)

[warning] Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)